tests/bson/bson-javascript_error-004.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+MongoDB\BSON\Javascript::__construct() does not allow PackedArray for scope
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+echo throws(function() {
+    new MongoDB\BSON\Javascript('function() { return 1; }', MongoDB\BSON\PackedArray::fromPHP([]));
```

Comment:
On a separate note, I noticed that many of the Javascript tests use `function foo()` as their code string. I don't think the function name is relevant or even correct, given how the server would invoke this, but it also doesn't affect our tests.

That may be worth cleaning up in a separate commit, though, just so we don't have bad example code laying around in the test suite.

Reply:
I agree. Feel free to create a low-hanging-fruits ticket if you don't want to make the change in this pull request.

Reply:
Added a separate commit to fix this and will merge once CI is green.